server/src/main/resources/c4.puml

```diff
@@ -1,5 +1,18 @@
 ' C4-PlantUML
-' https://github.com/plantuml-stdlib/C4-PlantUML/tree/v2.3.0
+' https://github.com/plantuml-stdlib/C4-PlantUML/tree/v2.4.0
```

Comment:
This line was not part of https://github.com/plantuml-stdlib/C4-PlantUML/blob/v2.4.0/C4.puml , but added for consistency.